src/Drush/Commands/PlaywrightDrushCommands.php

```diff
@@ -36,7 +38,7 @@ class PlaywrightDrushCommands extends DrushCommands {
    * @command test:node-clone
    * @aliases nc
    */
-  public function cloneNodeByTitle($node_type, $node_title, $new_node_title) {
+  public function cloneNodeByTitle($node_type, $node_title, $new_node_title, $moderation_state) {
```

Comment:
I would add a default value for moderation state here as well, even though we have it in the helper. Not a biggie

Reply:
done